docs/tutorials/fabric_remote.md

```diff
@@ -0,0 +1,255 @@
+
+For Firefly to succeesfully integrate and interact with your remote fabric network, you need to also deploy a Firefly batchpin chaincode in your network setup. This chaincode can be found in the [FireFly git repo](https://github.com/hyperledger/firefly/tree/main/smart_contracts/fabric/firefly-go).
+
+Once you have successfully packaged, installed, approved and commited this chaincode in your remote fabric network, carefully note down the chaincode name you gave and the channel name where this chaincode is deployed for future reference.
+
+> **NOTE:** You can deploy the batchpin chaincode to the same channel where your application chaincode lives or to a separate channel as well.
+
+### Update batchpin chaincode details in firefly core config
+
+In `$HOME/.firefly/stacks/my-remote-fabric/init/config`, you will find a firefly_core_0.yml file which is used by firefly core service in our firefly network setup.
+
+In this file, under the fabconnect section of blockchain-fabric section, you need to change the chaincode value to your batchpin chaincode name, and the channel value to the channel name where your batchpin chaincode was deployed in the previous step.
+
+### Starting your FireFly stack
+
+To start your stack now, simply run:
+
+```
+ff start my-remote-fabric
+```
+
+This may take a minute or two and in the background the FireFly CLI will do the following for you:
+
+- Download Docker images for all of the components of the Supernode
+- Set up configuration between all the components
+- Deploy FireFly's `BatchPin` smart contract to our remote fabric network
+- Register an identity for each member and node in our remote fabric network
+- It will probably also initialize a new blockchain and blockchain node running inside a container and also deploy an `ERC-1155` token smart contract to that local fabric network. This behaviour can be ignored.
+
+After your stack finishes starting it will print out the links to each member's UI and the Sandbox for that node. For this guide, it's just one member.
+
+```
+Web UI for member '0': http://127.0.0.1:5000/ui
+Sandbox UI for member '0': http://127.0.0.1:5108
+
+To see logs for your stack run:
+
+ff logs my-remote-fabric
+
+```
+
+## Integrate your FireFly stack with your Fabric chaincode 
+
+Once your stack is up and running, you can define and broadcast FireFly Interface Document for your remote fabric chaincode. You can refer to [Broadcast the Contract Interface(Fabric)](http://localhost:4000/firefly/tutorials/custom_contracts/fabric.html#broadcast-the-contract-interface) guide for this.
+
+You can also now create an HTTP API for your fabric chaincode which will help you easily query/invoke all your chaincode methods with your organizational context that you set up in fabconnect. You can refer to [Create an HTTP API for the contract(Fabric)](http://localhost:4000/firefly/tutorials/custom_contracts/fabric.html#broadcast-the-contract-interface) guide for this.
```

Comment:
```suggestion
You can also now create an HTTP API for your fabric chaincode which will help you easily query/invoke all your chaincode methods with your organizational context that you set up in fabconnect. You can refer to [Create an HTTP API for the contract(Fabric)](./custom_contracts/fabric.html#broadcast-the-contract-interface) guide for this.
```

These need to be relative links so they will link correctly when the docs site is published to GitHub pages.